bingo.js

```diff
@@ -84,10 +96,9 @@ $(document).ready(function()
 	// Set the background to a random image
 	document.body.style.backgroundImage = "url('Backgrounds/background" + (Math.floor(Math.random() * 10) + 1) + ".jpg')";
 
-	// By default hide the tooltips
+	// By default hide the tooltips and popups
 	$(".tooltip").hide();
-
-	$("#export").hide();
+	$(".popup").hide();
```

Comment:
This change will conflict with changes in commits c039779ea860082adfda79132d083348d2f64a02 and f04b0b7506ec412b32776757fa3515276b69525e from #117.

Reply:
To clarify, I changed the way tooltips and and export popup are displayed/hidden (with `display: none` by default) to avoid them flashing on loading of the website:

https://user-images.githubusercontent.com/624072/114309299-0233bc80-9ae7-11eb-82e5-fc6d284711e1.mp4

(recording was made using commit  dd81f1206ffc68da8c174891c087f575da8133a4)

Reply:
I've updated .popup to have default as `display: none`

Reply:
As it is right now, it shouldn't really 'conflict', but there will be unnecessary hiding/showing. Firstly mine will add class to popup to show it, and also changes in c039779 will add class to body to show the popup 🙂

Reply:
Regardless of semantic conflict, there definitely will be a textual merge conflict. To make the upcoming merge conflict resolution easier, I'll just drop commit c039779ea860082adfda79132d083348d2f64a02 from #117, since your rework of popup code is much more extensive. Side note: commit c039779ea860082adfda79132d083348d2f64a02 is off-topic, and not related to main feature of #117.

Aside from JS changes, there would also be conflict in `bingo.css` for the dark mode related changes (5455ede054b04a73bc13f207a5d35ba21d591c66).

Reply:
Actually, I'll drop the related commit f04b0b7506ec412b32776757fa3515276b69525e from my pull request as well.